lib/arke/exchange/base.rb

```diff
@@ -6,5 +8,9 @@ def initialize(config, strategy)
 
     def call(action)
     end
+
+    def push(action, data)
+      @strategy.push(Arke::Action.new(action, data))
```

Comment:
what is data? Action has params for it's data I guess

Reply:
It's a method for creating an `Arke::Action` from your data